### PR TITLE
P2P fixes 2

### PR DESCRIPTION
**Pull request by Martoon-00**

### CODE REVIEW

File: _docs/technical/protocols/2017-01-04-p2p.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ its neighbors (from the initial list of peers sent by the bootstrap node).
 A node sends messages to its peers, which resend messages to their peers close
 to the needed ID/key. The list of known peers is preserved between launches.
 
+Here and later, under **address** we mean tuple `(Host, Port, ID)`, while
```
Comment (Pastafarianist) [inline _docs/technical/protocols/2017-01-04-p2p.md]: s/under/by

File: _docs/technical/protocols/2017-01-04-p2p.md [hunk 2]
```diff
@@ -42,6 +42,9 @@ its neighbors (from the initial list of peers sent by the bootstrap node).
...
+**network address** would denote to just pair `(Host, Port)`.
```
Comment (Pastafarianist) [inline _docs/technical/protocols/2017-01-04-p2p.md]: s/would denote to just/denotes just the

File: _docs/technical/protocols/2017-01-04-p2p.md [hunk 3]
```diff
@@ -149,13 +153,14 @@ detection.
 
 ### Addressing
 
-For Kademlia addresses, we use the so-called HashNodeId. This makes assigning
-yourself an arbitrary ID impossible, and this means that a 100500 attack is the
-only way to perform an eclipse attack.
+As ID we use the so-called `HashNodeId`. Since it [contains](#id-structure)
+hash, assigning yourself an arbitrary ID is impossible, and this means that a
```
Comment (Pastafarianist) [inline _docs/technical/protocols/2017-01-04-p2p.md]: s/hash/a hash

File: _docs/technical/protocols/2017-01-04-p2p.md [hunk 4]
```diff
@@ -149,13 +153,14 @@ detection.
 
 ### Addressing
 
-For Kademlia addresses, we use the so-called HashNodeId. This makes assigning
-yourself an arbitrary ID impossible, and this means that a 100500 attack is the
-only way to perform an eclipse attack.
+As ID we use the so-called `HashNodeId`. Since it [contains](#id-structure)
```
Comment (Pastafarianist) [inline _docs/technical/protocols/2017-01-04-p2p.md]: "We use so-called `HashNodeId`s as node IDs."

File: _docs/technical/protocols/2017-01-04-p2p.md (hunk 3 above)
Comment (Martoon-00) [inline _docs/technical/protocols/2017-01-04-p2p.md]: Thanks for your review!

**Pull request merged**